core/controllers/custom_landing_pages_test.py

```diff
@@ -18,24 +18,42 @@
 import feconf
 
 
-class FractionLandingPageTest(test_utils.GenericTestBase):
-    """Test for showing landing page for fractions."""
+class FractionLandingRedirectPageTest(test_utils.GenericTestBase):
+    """Test for redirecting landing page for fractions."""
 
     def test_fraction_landing_page_without_viewer_type(self):
         """Test for showing the landing page for fractions,
         without any viewer type should redirect to teacher type.
         """
         response = self.get_html_response(
             feconf.FRACTIONS_LANDING_PAGE_URL, expected_status_int=302)
-        response.mustcontain('/fractions_landing/teachers')
+        self.assertIn('/learn/maths/fractions', response.headers['location'])
 
     def test_fraction_landing_page_with_viewer_type(self):
         """Test for showing the landing page for fractions,
-        with student viewer type should respond student type.
+        with student viewer type should redirect to student type.
```

Comment:
This outcome doesn't seem to be asserted...

Reply:
Added proper docstring.